scripts/create_dummy_profiles_json.py

```diff
@@ -0,0 +1,358 @@
+
+NOTE: the second file needs additional processing before importing.
+CustomerProfile needs to have UUID of the profile in the Helsinki profile service,
+and the UUID is generated by the Helsinki profile service at import time.
+When file (1) is imported, the Helsinki profile admin UI view returns a JSON file containing
+a mapping customer_id -> actual UUID
+
+api_url - use to fetch data about berths, piers and harbors. Running this script only reads data from the API,
+it does not change data in the server. The data is used to:
+* assign dummy customers to berths that are actually available
+* ensure dummy boats that fit in their berth
+
+-e flag can be used to specify the email address(es) to use for dummy users.
+If we actually send lots of emails while testing, then the
+recipient address should probably be a real email address, not something random.
+Mailgun (email service used as of 2020-12) has spammer detection logic and if it sees a lot of
+undeliverable mail, it might decide we are up to no good. "Mailgun has thresholds in place that if
+exceeded will result in a domain being temporarily disabled."
+
+Last name of the generated profiles starts with "Berth" so that the generated profiles
+can be identified in the database.
+
+"""
+
+import argparse
+import json
+import os
+import random
+import string
+import sys
+import traceback
+import uuid
+
+from faker import Faker
+from gql import Client, gql
+from gql.transport.requests import RequestsHTTPTransport
+
+NAME_CHARS = string.ascii_letters + "äöåÄÖÅ "
+EMAIL_CHARS = string.ascii_letters + "."
+
+fake = Faker("fi_FI")
+
+
+def randstr(chars=NAME_CHARS, min_len=1, max_len=10):
+    return "".join(
+        random.choice(chars) for _ in range(random.choice(range(min_len, max_len + 1)))
+    )
+
+
+HELSINKI_PROFILE_TEMPLATE = {
+    "customer_id": lambda: str(uuid.uuid4()),
+    "first_name": fake.first_name,
+    "last_name": lambda: "Berth" + fake.last_name(),
+    "email": randstr(min_len=10, max_len=10)
+    + "@"
+    + randstr(min_len=10, max_len=10)
+    + ".com",
+    "address": {
+        "postal_code": fake.postcode,
+        "address": fake.street_address,
+        "city": fake.city,
+    },
+    "phones": [fake.phone_number],
+}
+
+COMPANY_TEMPLATE = {
+    "type": "company",
+    "name": fake.company,
+    "address": fake.address,
+    "postal_code": fake.postcode,
+    "city": fake.city,
+}
+
+
+def generate_organization():
+    if random.random() > 0.9:
+        return COMPANY_TEMPLATE
+    else:
+        return None
+
+
+# place for variables that are used while applying a template
+template_context = {}
+
+LEASE_TEMPLATE = {
+    "harbor_servicemap_id": lambda: template_context["berth"]["harbor_servicemap_id"],
+    "pier_id": lambda: template_context["berth"]["pier_identifier"],
+    "berth_number": lambda: int(template_context["berth"]["number"]),
+    "start_date": "2020-06-10",
+    "end_date": "2020-09-14",
```

Comment:
This is a bit tricky... maybe we can parametrise the season. We take `season year` as parameter, and we default it to the current year. Then, we can use the [leases/utils.py](https://github.com/City-of-Helsinki/berth-reservations/blob/master/leases/utils.py) functions to calculate start and end dates:
```python
from lease.utils import calculate_season_start_date, calculate_season_end_date
LEASE_TEMPLATE = {
    "start_date": calculate_season_start_date(season_year),
    "end_date": calculate_season_end_date(season_year),
}
```

Reply:
fixed.